ajax/libs/jquery.form/package.json

```diff
@@ -2,41 +2,23 @@
   "name": "jquery.form",
   "filename": "jquery.form.min.js",
   "title": "jQuery Form Plugin",
-  "description": "A simple way to AJAX-ify any form on your page; with file upload and progress support.",
-  "keywords": [
-    "form",
-    "upload",
-    "ajax"
-  ],
-  "version": "3.51",
```

Comment:
With the autoupdate feature, doesn't cdnjs pull the latest release from the github repo? If this `version` field exists, am I going to need to submit a new pull request every time the library is updated?